docs/add_doc.md

```diff
@@ -2,7 +2,7 @@
 
 For each new page in the docs, you will need to add `.md` file to the `docs` folder and then add a route to this page in `mkdocs.yml` (under `nav`).
 
-To build this locally, you will need to download the docs requirements, `pip install -r docs/requirements.txt`.
+To build this locally, you will need to download the docs requirements by running `pip install -r docs/requirements.txt` followed by `mkdocs serve`.
```

Comment:
Since it can now be accessed via a website, could this page be removed?

Reply:
I think we should keep it because the website very slow to show the updates so it's much better to show it locally (it can help for future maintain of the documentation) What do you think?

Reply:
okay that is fair